src/lib.rs

```diff
@@ -442,11 +445,31 @@ pub trait Rng {
     /// use rand::{thread_rng, Rng};
     ///
     /// let mut v = [0u8; 13579];
-    /// thread_rng().fill_bytes(&mut v);
+    /// thread_rng().fill(&mut v);
     /// println!("{:?}", &v[..]);
     /// ```
-    fn fill_bytes(&mut self, dest: &mut [u8]) {
-        impls::fill_bytes_via_u64(self, dest)
+    fn fill(&mut self, dest: &mut [u8]) {
+        impls::fill_via_u64(self, dest)
+    }
+
+    /// Fill `dest` entirely with random data.
+    ///
+    /// Implementations of this trait must implement at least one of
+    /// `next_u32`, `next_u64`, `fill` and `try_fill` directly. In the case this
+    /// function is not implemented directly, the default implementation will
+    /// generate values via `fill`.
+    /// 
+    /// This is the only method which allows an RNG to report errors while
+    /// generating random data; other methods either handle the error
+    /// internally or panic. This method is intended to allow error-handling
+    /// when seeding keys or PRNGs from external (true) RNGs.
```

Comment:
Sorry for being picky... "This method is intended to" sounds a bit specific. Maybe "This method can be used to"?

Reply:
It is specific, and intended to explain why there isn't `try_next_u64` etc. You think it's over-specific? I mean it's obvious there *may* be other ways to use it, but this is the only important one I see.

Reply:
That is a good intention, useful to add. Could you reword it a little, because I missed it?
Something like

> This method is intended to be the way to use external (true) RNGs, like `OsRng`. You primarily want to use it to seed keys or (infallible) PRNGs.

Reply:
Okay, done. I'll rebase later, to let you review changes now.

Reply:
I am not sure what there is left to rebase or review?